react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+// const numbers = [1, 2, 3, 4, 5];
+const listItems = pokedex.map(pokedex =>
+  <li key={pokedex} value={pokedex.name}>{pokedex.name}</li>
```

Comment:
The `key` should use the `number` property and you don't need a `value` attribute on a `li`:
```JSX
<li key={pokedex.number}>{pokedex.name}</li>
```